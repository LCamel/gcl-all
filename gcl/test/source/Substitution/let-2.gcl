var a : Int
{: P x = a = x :}

a := 3

{ P a }