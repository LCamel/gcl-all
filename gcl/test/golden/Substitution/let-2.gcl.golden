1: P [ a \ 3 ] 3
  P [ a \ 3 ] 3 ===> 3 = 3
    
2: P [ a \ 3 ] 3
  P [ a \ 3 ] 3 ===> 3 = 3
    